tests/test_qfit_ligand.py

```diff
@@ -19,11 +19,10 @@ class TestQFitLigand:
     def mock_main(self):
         # Prepare args
         args = [
-            "../example/composite_omit_map.mtz",  # mapfile, using relative directory from tests/
-            "../example/4ms6.pdb",  # structurefile, using relative directory from tests/
+            "./example/composite_omit_map.mtz",  # mapfile, using relative directory from tests/
+            "./example/4ms6.pdb",  # structurefile, using relative directory from tests/
             "-l", "2FOFCWT,PH2FOFCWT",
-            "A, 702",  # selection
-            "--directory", f"{os.environ['QFIT_OUTPUT_DIR']}",
+            "A, 702"  # selection
```

Comment:
I would recommend leaving a trailing comma here on this last list item.
Doing so means that if someone comes and adds a new arg later on, they do it all on a single new line, they don't need to edit this line. (This helps massively with rebase/merge conflicts).
```suggestion
            "A, 702",  # selection
```